### ISSUE TITLE
Un-installing the module leaves the default admin page with no valid process.

### ISSUE THREAD

**Issue by netcarver** [Notanotherdotcom/ProcessDashboard#1]
After un-installing the module, the default admin page no longer has a valid process assigned, leading to a notice on the screen after login. Perhaps an `uninstall()` method is needed that re-instates the initial value of the root Admin page's process?

**Comment by Notanotherdotcom** [comment]
Good call, took it a few steps further too: https://github.com/Notanotherdotcom/ProcessDashboard/commit/fef7c1249a02d1e99df365ff0bb406ff95f3c398
